### PR TITLE
Kaduk changes

### PR DESCRIPTION
**Pull request by martinduke**

### CODE REVIEW

File: proposed_charter.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ control and loss recovery algorithms are out of scope.
 Multicast UDP and multicast IP support are out of scope. However, the group may
 specify extension points that would enable future work on multicast. Specifying
 proxy server discovery mechanisms is also out of scope, but the group may
-specify techniques for identifying proxy servers to aid future discovery
-mechanisms.
+specify identifiers and capability reports for proxy servers to aid future
```
Comment (chris-wood) [inline proposed_charter.md]: Hmm... can we revert this? I don't know what an identifier or capability report is. The previous text was pretty clear, in my opinion.

**Review by chris-wood — CHANGES_REQUESTED** [review]
Requesting changes due to the discovery-related text.

File: proposed_charter.md [hunk 2]
```diff
@@ -7,26 +7,26 @@ by hiding a client's IP address from a target server.
...
 CONNECT is currently limited to TCP. In contrast, HTTP/3 is a viable candidate
-protocol for proxying arbitrary traffic, as it provides secure connectivity,
-multiplexed streams, and migration for a single connection while taking
-advantage of a unified congestion controller. An HTTP/3 datagram construct built
-on top of QUIC datagram frames would provide for unreliable data transmission
-and enable transporting UDP and other unreliable flows via a proxy. Moreover, it
-would not introduce potentially redundant or unnecessary recovery mechanisms.
-Lastly, HTTP supports an established request/response semantic that can set up
-and configure flows for different services.
+protocol for proxying arbitrary traffic, as it provides to a single connection
```
Comment (erickinnear) [inline proposed_charter.md]: I see how this is trying to be different for the comment, but `provides to a single connection` reads oddly to me

Comment (erickinnear) [reply proposed_charter.md]: Trying to think of some suggested text

File: proposed_charter.md (hunk 1 above)
Comment (erickinnear) [inline proposed_charter.md]: What's a "capability report" in this context? Like a way to advertise the things you support, I assume?

File: proposed_charter.md [hunk 3]
```diff
@@ -35,12 +35,12 @@ control and loss recovery algorithms are out of scope.
...
 proxy server discovery mechanisms is also out of scope, but the group may
-specify techniques for identifying proxy servers to aid future discovery
-mechanisms.
+specify identifiers and capability reports for proxy servers to aid future
+discovery mechanisms.
 
 Impacts on address migration, NAT rebinding, and future multipath mechanisms of
 QUIC are not anticipated. However, the working group should document these
-impacts if they arise.
+impacts, or those of any other QUIC developments, if they arise.
```
Comment (erickinnear) [inline proposed_charter.md:43]: 👍

**Review by erickinnear — COMMENTED** [review]
Thanks for writing up these changes!

File: proposed_charter.md (hunk 1 above)
Comment (martinduke) [reply proposed_charter.md]: will reword in the next commit

Comment (martinduke) [reply proposed_charter.md]: will reword in the next commit.

File: proposed_charter.md (hunk 2 above)
Comment (martinduke) [reply proposed_charter.md]: updated

**Review by chris-wood — APPROVED** [review]
LGTM!

**Review by erickinnear — APPROVED** [review]
LGTM, thanks @martinduke!

**Review by DavidSchinazi — COMMENTED** [review]
Looks like I rode in after the battle ended, but FWIW I approve this change :)

**Pull request merged**